web/src/message_notifications.js

```diff
@@ -94,9 +94,6 @@ function get_notification_title(message, msg_count) {
     }
 
     switch (message.type) {
-        case "test-notification":
-            other_recipients = remove_sender_from_list_of_recipients(message);
-            break;
```

Comment:
This is this unused? I think this is the code path for the "test desktop notifications" button in settings.

Reply:
Yeah, I tested it would this change and this is not used.

Reply:
OK, and indeed I don't see what this would even do, reading the code.